packages/ui-predicate-react/src/ui-predicate.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from 'react';
+import fs from 'fs';
+import UIPredicateCoreReact from './UIPredicateCoreReact';
+import { InitializationFailed } from './errors';
+import UIPredicateCompound from './ui-predicate-compound';
+import './ui-predicate-react.css';
```

Comment:
Mardii, i don't know well ReactJS in a browser context so, why did you import style here and then read the file content and appending it to a new node ?